src/LaunchDarkly.EventSource/Configuration.cs

```diff
@@ -129,7 +149,8 @@ public TimeSpan MaximumDelayRetryDuration
         ///     <p>- or - </p>
         ///     <p><paramref name="readTimeout"/> is less than zero. </p>
         /// </exception>
-        public Configuration(Uri uri, HttpMessageHandler messageHandler = null, TimeSpan? connectionTimeOut = null, TimeSpan? delayRetryDuration = null, TimeSpan? readTimeout = null, IDictionary<string, string> requestHeaders = null, string lastEventId = null, ILog logger = null)
+        public Configuration(Uri uri, HttpMessageHandler messageHandler = null, TimeSpan? connectionTimeOut = null, TimeSpan? delayRetryDuration = null, TimeSpan? readTimeout = null, IDictionary<string, string> requestHeaders = null, string lastEventId = null, ILog logger = null,
```

Comment:
I had to remind myself that C# has a language feature where constructor parameters with defaults do not need to be specified, which is why we don't have a builder here.

Reply:
Yeah, I'm not crazy about how this looks either. I would rather have a builder. But then our fork would be diverging quite a bit for reasons unrelated to functionality.